test_modules/package.json

```diff
@@ -1,4 +1,5 @@
 {
   "name": "webpack-canary-test",
-  "version": "0.0.1"
+  "version": "0.0.1",
```

Comment:
What specifically? The file or the info in it?
The file is needed so that npm knows it should install modules here, and not in the parent folder.
The info is not technically required, but it prevents some  yarn warnings. This could probably be removed once "Support yarn warnings in stderr" from #31 is implemented.